wandb/sdk/launch/agent/__init__.py

```diff
@@ -104,20 +114,28 @@ def run_job(self, job):
         print("agent: got job", job)
         # parse job
 
-        
         # todo: this will only let us launch runs from wandb (not eg github)
         run_spec = job["runSpec"]
-        entity = run_spec["entity"]
-        project = run_spec["project"]
-        run_id = run_spec["run_id"]
+        wandb_entity = run_spec["entity"]
+        wandb_project = run_spec["project"]
         resource = run_spec["resource"]
         entry_point = run_spec["overrides"].get("entrypoint")
-        uri = "{}/{}/{}/runs/{}".format(self._base_url, entity, project, run_id)
+        uri = run_spec["uri"]
```

Comment:
lol good catch

Reply:
lmao, some of my initial work may have been that early morning work